experimental/yarpl/src/yarpl/flowable/sources/Flowable_FromObservable.h

```diff
@@ -122,22 +104,23 @@ class FlowableFromObservableSubscription
    */
   class TheObserver : public yarpl::observable::Observer<T> {
```

Comment:
That's a good question. I typically default to composition rather than inheritance as I find it cleaner most of the time, but I'll give this a try as you suggest.